packages/qdrant-loader/src/qdrant_loader/core/worker/pool.py

```
@@ -102,7 +102,58 @@ async def worker() -> None:
+                                await asyncio.sleep(current_renewal_interval)
+                                try:
+                                    async with self._queue_io_guard:
+                                        extended = await self._queue.extend_visibility(
+                                            current_job_id,
+                                            current_lease_seconds,
+                                            claim_attempt=current_claim_attempt,
+                                        )
+                                    if not extended:
+                                        lease_lost_event.set()
+                                        return
+                                except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)